openssl-sys/build/main.rs

```diff
@@ -199,6 +199,7 @@ See rust-openssl README for more information:
             (8, 1) => ('8', '1'),
             (8, _) => ('8', 'x'),
             (9, 0) => ('9', '0'),
+            (9, 1) => ('9', '1'),
```

Comment:
The release notes say that 2.9.1 is the first stable release for the 2.9 series, so we can whitelist future 2.9.x versions.